backend/src/main/java/gov/cdc/usds/simplereport/service/PatientLinkService.java

```diff
@@ -83,4 +72,10 @@ public PatientLink refreshPatientLink(String internalId) {
     pl.refresh();
     return plrepo.save(pl);
   }
+
+  public PatientLink expireMyPatientLink() {
+    PatientLink pl = contextHolder.getPatientLink();
+    pl.expire();
+    return plrepo.save(pl);
+  }
```

Comment:
This is mostly reflective of my ignorance of Spring DI, but does the presence of the `contextHolder` property limit the lifetime of each `PatientLinkService` instance to a single request? It looks like the class would otherwise be usable as a singleton, since all other methods take identifiers and don't read from `contextHolder`.

Reply:
I can take this one. 😁 

The `CurrentPatientContextHolder` bean and the `PatientLinkService` bean are both singletons: the latter is totally normal, the former is actually a proxy for a thread-bound object that is scoped to the request.